app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+    for human in people:
+        person_instance = Person.people[human["name"]]
+        if human.get("wife"):
+            person_instance.wife = Person.people[human["wife"]]
+        if human.get("husband"):
+            person_instance.husband = Person.people[human["husband"]]
```

Comment:
Accessing `Person.people[human["husband"]]` assumes that the husband exists in the dictionary. Consider adding error handling to manage cases where the husband might not be found.